docs/community/20230104-mixed-clusters.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Feature Group for Mixed Clusters in Cluster API
+authors:
+  - "@prometherion"
+  - "@lentzi90"
+reviewers:
+  - "@lentzi90"
+  - "@richardcase"
+creation-date: 2023-01-04
+last-updated: 2023-01-06
+status: proposed
+see-also:
+  - https://github.com/kubernetes-sigs/cluster-api/issues/7475#issuecomment-1302454745
+  - https://kubernetes.slack.com/archives/C8TSNPY4T/p1667458956589639
+---
+# Mixed Clusters in Cluster API Feature Group
+
+This document briefly outlines the scope, communication media, and stakeholders for a formal Feature Group dedicated to defining a Cluster API-approved solution for "mixed clusters" which combines two or more infrastructure providers.
+
+## User Story and Problem Statement
+
+As a Kubernetes cluster administrator, I want the ability to specify different infrastructure providers for the Control Plane and the Worker nodes to decouple each-other infrastructure providers.
+
+At the current state, Cluster API is giving for granted that worker nodes and control plane components are backed by the same infrastructure provider. However, this is a hard statement, since the two components could be backed by different infrastructure providers.
+
+A practical example is offered with [Kamaji](https://github.com/clastix/kamaji) that offers virtualized Control Plane resources used by worker nodes provisioned on any cloud provider by using the `kubeadm` bootstrapper.
```

Comment:
Being this a working group I think we need more examples or use cases of mixed clusters from different companies

Reply:
@richardcase maybe you can help with something here?
I would like to highlight that Kamaji provides a *containerized* control plane running in a *pre-existing* kubernetes cluster.
Another (quite different) example would be to combine a baremetal provider like Metal3 with virtualized providers. For example, CAPO for control plane and auto scaling node pool + Metal3 for baremetal needs.